pootle/urls.py

```diff
@@ -30,7 +31,15 @@
     url(r'^accounts/evernote/', include('evernote_auth.urls')),
     url(r'^admin/reports/', include('evernote_reports.urls')),
     url(r'', include('evernote_reports.profile_urls')),
+)
+
+if settings.POOTLE_ENABLE_OFFLINE:
```

Comment:
By that I meant ``app/import_export/urls.py`` but maybe that is the ideal world, but I thought it was what we were striving for here and I/E is an app where we slowly push towards that goal.